interface/usergroup/usergroup_admin_add.php

```diff
@@ -351,9 +353,59 @@ function authorized_clicked() {
    }
   ?>
   </select></td>
+  
   <td><span class="text"><?php echo xlt('Additional Info'); ?>: </span></td>
   <td><textarea name=info style="width:120px;" cols=27 rows=4 wrap=auto></textarea></td>
+  </tr>
+  <tr>
+  <td><span class="text"><?php echo xlt('Full-screen role:'); ?></span></td>
```

Comment:
move the hyphen and colon outside of the translation ( xlt() )

Reply:
Okay